### PR TITLE
Replace `os.system` with `subprocess`.

### PR DESCRIPTION
**Pull request by upskaling**
Avoid calling `/bin/sh` to execute the command.

### CODE REVIEW

File: speech/audioutils.py [hunk 1]
```diff
@@ -37,20 +31,23 @@ def get_audio_commands(text, outfile, lang, cache_path, speed):
             # low the limits to avoid overflow
             or len(text) + len(discours[idx + 1]) >= overflow_len
         ):
-            filename = cache_path + 'speech' + str(idx) + '.wav'
+            filename = cache_path + '/speech' + str(idx) + '.wav'
```
Comment (mothsART) [inline speech/audioutils.py:34]: this change is good point but need a specially pull request for this bugfix 

save tmp file on ~/.cache/gSpeech/speech_x.wav and not ~/.cache/gSpeechspeech_x.wav

File: speech/audioutils.py [hunk 2]
```diff
@@ -15,16 +15,10 @@ def get_audio_commands(text, outfile, lang, cache_path, speed):
...
-    text = text.replace('"', '')
-    text = text.replace("'", '')
     # low the limits to avoid overflow
     if len(text) <= overflow_len:
-        stream = """pico2wave -l %s -w %s '%s'""" % (
-            lang,
-            outfile,
-            effect(text, speed * 100)
-        )
+        stream = ['pico2wave', '-l', lang, '-w', outfile, '--',
```
Comment (mothsART) [inline speech/audioutils.py:20]: You can delete stream variable, nonsense anyway

File: speech/audioutils.py [hunk 3]
```diff
@@ -15,16 +15,10 @@ def get_audio_commands(text, outfile, lang, cache_path, speed):
     overflow_len = 30000
     cmds = []
     names = []
-    # remove parenthesis to avoid bugs with pico2wave command
-    text = text.replace('"', '')
-    text = text.replace("'", '')
```
Comment (mothsART) [inline speech/audioutils.py:20]: can you remove this 3 lines and send it to another PR. string pass to picowave seems not exactly the same

Comment (upskaling) [reply speech/audioutils.py:20]: It seems unnecessary to me with subprocess

Comment (mothsART) [reply speech/audioutils.py:20]: double quote replacing is probably useless because treated previously but simple quote not.
I don't undersand relationship with subprocess.
